cxx-sensors/src/main/java/org/sonar/cxx/sensors/utils/CxxIssuesReportSensor.java

```diff
@@ -143,10 +143,11 @@ private InputFile getInputFileTryRealPath(SensorContext sensorContext, String pa
     Path realPath;
     try {
       realPath = absolutePath.toRealPath(LinkOption.NOFOLLOW_LINKS);
-    } catch (IOException e) {
+      // despite of API the RuntimeExceptions can be thrown too

```

Comment:
@ivangalkin this comment has no additional value

Reply:
true, done